pkgs/applications/graphics/normcap/default.nix

```diff
@@ -0,0 +1,43 @@
+# From NUR https://github.com/nix-community/nur-combined/blob/6bddae47680482383b5769dd3aa7d82b88e6cbc8/repos/renesat/pkgs/normcap/default.nix
+
+{
+  lib,
+  stdenv,
+  python3,
+  fetchFromGitHub,
+}:
+python3.pkgs.buildPythonApplication rec {
+  pname = "normcap";
+  version = "0.4.4";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "dynobo";
+    repo = "normcap";
+    rev = "v${version}";
+    hash = "sha256-dShtmoqS9TC3PHuwq24OEOhYfBHGhDCma8Du8QCkFuI=";
+  };
+
+  nativeBuildInputs = with python3.pkgs; [
+    poetry-core
+  ];
+
+  propagatedBuildInputs = with python3.pkgs; [
+    pyside6
+
+    # Test
+    toml
+    pytest-qt
+  ];
+
+  postPatch = ''
+    sed -i 's|PySide6-Essentials = "6.5.1"||' pyproject.toml
```

Comment:
Use `substituteInPlace` instead.
https://nixos.org/manual/nixpkgs/stable/#fun-substitute

Reply:
I tried in the last push, can you check that it's correct?

Reply:
Yes, perfect !